### PR TITLE
remove transition test which was only necessary for ie8, which is no …

### PR DESCRIPTION
**Pull request by paxperscientiam**
…longer supported

Given that `readme` indicates that ie8 is no longer supported, i got rid of the transition test. It's not needed for ie versions > 8 , right?

### CODE REVIEW

File: src/imagelightbox.js [hunk 1]
```diff
@@ -542,9 +528,6 @@
                             if (!wasTouched(e.originalEvent) || options.quitOnImgClick) {
                                 return true;
                             }
-                            if (hasCssTransitionSupport) {
-                                imagePosLeft = parseInt(image.css('left'));
-                            }
```
Comment (rejas) [inline src/imagelightbox.js]: was that check wrong or why did you delete the line?

**Comment by rejas** [comment]
According to https://caniuse.com/#feat=css-transitions IE9 doesnt support it either. Then again, should we really still support that old stuff? Maybe get rid of IE below 11 for a v1 ? what are your thoughts @paxperscientiam  @GenaBitu  ?

**Comment by GenaBitu** [comment]
Ok, let's talk browser support:

I would really like to see some automated browser support testing as part of the lints. I am familiar with amilajack/eslint-plugin-compat (for eslint) and ismay/stylelint-no-unsupported-browser-features (for stylelint) - there should be something similar to that for csslint as well.
Will try to add a PR for this if we reach some conclusion.

I tend to support even IE8, I generally use the lints as a guideline and use [cover 95%, not < 0.1%](https://browserl.ist/?q=cover+95%25%2C+not+%3C+0.1%25) as the preferred coverage setting.

I think this discussion is largely subjective (i. e. you see a lot of arguments about this...). I agree that IE8 should die, but the fact is that some people still use it and I want my sites to work for as many people as possible. Also, bear in mind that in some countries, the browser usage chart looks vastly different.

To sum it up (as if this comment wasn't long enough...): 1) We should talk browser coverage, not individual browsers. 2) We should have automatic lints. 3) I support the aforementioned coverage "query".

File: src/imagelightbox.js (hunk 1 above)
Comment (paxperscientiam) [reply src/imagelightbox.js]: stupidity

**Comment by paxperscientiam** [comment]
These days, projects include pre-built files under "dist" that are meant to be ready for consumption. Since this repo includes source files, any developer is free to choose to ingest them into their own build process.

In short, if your preferred compatibility profile for your dist files is insufficient for some dev, they are free to use the source files.

To put it another way, the "dist" files are a mere convenience and not guaranteed to meet every developer's needs.

Does that help?

File: src/imagelightbox.js (hunk 1 above)
Comment (paxperscientiam) [reply src/imagelightbox.js]: @rejas Actually, I'm confused. That bit of code is defined for when `hasCssTransitionSupport` is truthy, but the `imagePosLeft` is used for when `hasCssTransitionSupport` is falsey. What gives?

Comment (rejas) [reply src/imagelightbox.js]: It's like this already in the original code. I am wondering how it would look like if we change this line....

**Comment by marekdedic** [comment]
Hi,
this should be fixed as of #870, let me know if it isn't :) Thanks!
